packages/ibm-products/src/components/Datagrid/Datagrid/DatagridExpandedRow.tsx

```diff
@@ -37,6 +37,9 @@ const DatagridExpandedRow =
       <tr
         className={cx(`${blockClass}__expanded-row`, {
           [`${blockClass}__slug--row`]: isValidElement(row?.original?.slug),
+          [`${blockClass}__aiLabel--row`]: isValidElement(
+            row?.original?.aiLabel
+          ),
```

Comment:
```suggestion
          ) || isValidElement(row?.original?.slug),
```

Reply:
Since some adapters are already using the existing slug classes, completely changing the classnames would break their code. Hence a separate class is added for AiLabel. Once we stop the support for slug completely, we could just remove the slug classes. For now it would be safer to keep separate classes.